guests/invitation.py

```diff
@@ -36,12 +36,15 @@ def get_invitation_context(party):
     }
 
 
-def send_invitation_email(party, test_only=False, recipients=None):
+def send_invitation_email(party, test_only=False, recipients=None, unique_addresses_only=False):
```

Comment:
is there any world in which you want this disabled? it seems sensible to me to always remove them, since sending a mail to the same address twice is never intentional...